benchexec/tools/cetfuzz.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    cetfuzz
+    """
+
+    def executable(self, tool_locator):
+        
+        return tool_locator.find_executable("runTool.py", subdir=".")
+
+    def version(self, executable):
+        return self._version_from_tool(executable, use_stderr=True)
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
```

Comment:
Why are you overwriting this? Unless `REQUIRED_PATHS` is set it will not do anything.